state-manager-frontend/README.md

```
@@ -0,0 +1,198 @@
+- **NEXT_CREATED**: Next state has been created based on this execution
+- **RETRY_CREATED**: Retry state has been created due to failure
+- **TIMEDOUT**: State execution timed out
+- **ERRORED**: State execution failed with an error
+- **CANCELLED**: State execution was cancelled
+- **SUCCESS**: State completed successfully
+
+## Getting Started
+
+### Prerequisites
+
+- Node.js 18+ 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)